react/features/invite/components/DialInNumbersForm.js

```diff
@@ -1,7 +1,6 @@
 import Button from '@atlaskit/button';
-import { DropdownMenuStateless as DropdownMenu } from '@atlaskit/dropdown-menu';
-import { FieldTextStateless as TextField } from '@atlaskit/field-text';
-import ChevronDownIcon from '@atlaskit/icon/glyph/chevron-down';
+import DropdownMenu, {
+    DropdownItem, DropdownItemGroup } from '@atlaskit/dropdown-menu';
```

Comment:
Refactored the DialInNumbersForm to use the latest DropdownMenu techniques and removed the deprecated functions and properties.